### ISSUE TITLE
Auto spell check not working for Gmail

### ISSUE THREAD

**Issue by kcinman11358** [wavebox/waveboxapp#954]
* Wavebox Version: 4.7.3
* Operating System & Version: Windows 10
* Account type (if applicable): Gmail

**Expected & actual behavior**
Expected: red line appears under misspelled words
Actual: No red line

![image](https://user-images.githubusercontent.com/7409848/53523321-24525800-3aa2-11e9-82d8-7c9a39570c63.png)

**Steps to reproduce**
Type a wrong wrong and see

**Is the bug persistent or intermittent?**
Intermittent, but is happening more often

**Screenshots**
Screenshots often help us narrow down a bug more easily.
![image](https://user-images.githubusercontent.com/7409848/53523337-2caa9300-3aa2-11e9-8a24-8e1752200bcd.png)

**Additional information**

**Comment by nmat** [comment]
hmm just tested it here and I see red lines under the lines

![image](https://user-images.githubusercontent.com/383031/53525154-6f3f9000-3ae1-11e9-9343-19374e566416.png)

**Comment by nmat** [comment]
Can you check what you have selected with:

![image](https://user-images.githubusercontent.com/383031/53525293-b299fe80-3ae1-11e9-897b-d9b90b4f5246.png)

inside wavebox?

**Comment by kcinman11358** [comment]
Yes, I have this checked. It was working but since this latest update, I think it stopped
![image](https://user-images.githubusercontent.com/7409848/53526357-937f7a80-3aa9-11e9-9113-45204f2e26dd.png)

**Comment by nmat** [comment]
do you say it happens sometimes?

What happens if you restart wavebox does it work after that?

**Comment by kcinman11358** [comment]
Restarting seems to have fixed it now. Thanks.

I'll close the issue

**Comment by nmat** [comment]
Awesome! No problem. :)

**Comment by Thomas101** [comment]
Out of curiosity, do you have Wavebox set to start at system startup?

**Comment by kcinman11358** [comment]
@Thomas101 yes i do

**Comment by Thomas101** [comment]
Thanks - I think it maybe down to a timeout. I've already added https://github.com/wavebox/waveboxapp/commit/ec528c7c7ce039cb124eaea18f0fd521d55420c1 which I think should help with this :)

**Comment by kcinman11358** [comment]
Awesome! Thank you so much @Thomas101 !!

**Comment by kcinman11358** [comment]
I'm reopening this issue because it seems to have gotten worse with the past few updates. I am having to reboot wavebox a lot and even then it doesn't fix it.

@Thomas101, is there something more you need to help you triage?

**Comment by Thomas101** [comment]
Thanks for the update @kcinman11358 

It would be good to know a little bit more about your machine and setup...
* Which CPU you have and how much RAM do you have?
* Do you know if you're using a solid state drive (SSD) or traditional hard drive?
* Have you noticed if the behaviour more broken if the app launches on startup at all?

There's a setting under Settings > Advanced that might be worth changing - just to see if it makes any difference. It's called `In process spellchecking`, try turning this off to see if that helps

**Comment by kcinman11358** [comment]
Sorry for the delay
CPU: Processor	Intel(R) Core(TM) i7-6500U CPU @ 2.50GHz, 2592 Mhz, 2 Core(s), 4 Logical Processor(s)
RAM: 16gb
I have a SSD
It seems to be better now since latest update. If it starts acting up again, I'll try this. Thanks

**Comment by Thomas101** [comment]
Thanks, keep me up to date. The spellcheck library and runtimes were updated in 4.8.4 so hopefully this has fixed :)

**Comment by kcinman11358** [comment]
The spell check seems to be getting worse. Even after restarting it, the issue still remains the majority of the time. Sometimes it comes back without restarting, but it will go away eventually too.

**Comment by Vafa-Andalibi** [comment]
Having the same problem with spell check on Linux 18.04 (unity). Restarting Wavebox fixes it but it happens occasionally and I'm not sure what triggers it.

**Comment by Thomas101** [comment]
@Vafa-Andalibi do you know how you installed Wavebox (e.g. through the snap store/apt repository) & also which version of Wavebox you're using? Do you have the app set to launch on system start?

**Comment by Vafa-Andalibi** [comment]
@Thomas101 Sorry I don't use wavebox anymore so I can't check the versions. 
But I had it on 3 computers Macbook Air, iMac, and Ubuntu 18.04. 
In Ubuntu I installed it with snap, had the same problem, uninstalled it and installed it with the latest released binary, still the same.
In Macbook and iMac I downloaded the latest release binary and then updated them a few times when new versions were released. 
I had this problem on all three computers.

**Comment by rubiworks** [comment]
Same here....with all of the above issues.

**Comment by rubiworks** [comment]
@Thomas101  Windows 10 Pro 64bit

**Comment by lestevam** [comment]
I also have problems with spell checking in KDE Neon 5.16. But my problem is that the language defined for PT-BR and the wave does not consider the language and always try to perform the check for EN.

**Comment by Mooz80** [comment]
Hi,
I do have the same issue here with Gmail and spellcheck stopping to work intermittently. 
OS: Mac 10.14.6 (18G95)
16 GB RAM
CPU: 2 GHz Intel Core i5
Wavebox Version 4.11.3 (dmg)

Restarting usually works (2-3 times a day), but today I had to disable spell check completely and re-enable it. 

Anything I could gather for you that could help resolve the issue?

**Comment by Mooz80** [comment]
<img width="731" alt="Sample" src="https://user-images.githubusercontent.com/18400396/65265239-73171c00-dade-11e9-8b17-e8967879229e.png">

See a sample of the problem today. still not working even after restarting the application...

**Comment by Thomas101** [comment]
@Mooz80 there are a couple of things that would be good to know, we're trying to figure out what the common use-case is here so we can get to the bottom of it.

1. Do you have any extensions installed?
2. What does your `Settings` > `Language` section look like?
3. Is it just Gmail that's affected, or do you see the same on other sites?
4. Do you ever find the case where you start the app and spellcheck works, then after some time it stops? Or is it that spell check doesn't work from when the app is launched.

**Comment by Mooz80** [comment]
@Thomas101 
Hi, thanks for you follow-up. Here are a few answers:  
> 1. Do you have any extensions installed? _- No extensions installed_
> 2. What does your `Settings` > `Language` section look like? - _see image below_
> 3. Is it just Gmail that's affected, or do you see the same on other sites? _- I only use it for my gmail accounts now. I will connect an Outlook account and see the behaviour next time it happen and keep you posted._
> 4. Do you ever find the case where you start the app and spellcheck works, then after some time it stops? Or is it that spell check doesn't work from when the app is launched.
_First choice: spell check works for a while, then stops. Closing the app and reopening quickly does not seem to work all the time, like if I need to close it for more than 5 seconds for it to work back._ 

<img width="531" alt="Spellcheck" src="https://user-images.githubusercontent.com/18400396/65327011-c772d800-db80-11e9-8d7a-b78e9e0b7152.png">

**Comment by Mooz80** [comment]
@Thomas101 Hi again
Problem occurred again and I checked if only Gmail was affected. Here is the situation I have right now for my 3 accounts: 
Account 1: Gmail - Spellcheck stopped working
Account 2: Gmail - Spellcheck still working
Account 3: Outlook - Spellcheck still working. 

So, it looks like it's not Gmail only, but linked to specific account as well. 
Thanks for your support.

<img width="461" alt="spellcheck2" src="https://user-images.githubusercontent.com/18400396/65352681-3d933100-dbb9-11e9-9f0e-d4708dc02bb8.png">
